defaults/main.yml

```diff
@@ -5,20 +5,26 @@
 consul_debug: no
 
 ### Package
-consul_version: "{{ lookup('env','CONSUL_VERSION') | default('0.8.2', true) }}"
+consul_version: "{{ lookup('env','CONSUL_VERSION') | default('0.8.3', true) }}"
 consul_architecture_map:
   # this first entry seems... redundant
   #(but it's required for reasons)
   amd64: amd64
   x86_64: amd64
   armv7l: arm
   aarch64: arm64
+  32-bit: "386"
```

Comment:
I don't know if there are even Windows Server versions that run in 32-bit, but there are personal versions of Windows that do, and this entry ensures the right package is grabbed from Hashicorp.